script/install_04_kubernetes_master_ingress.yaml

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+ingress_http_port="80"
+ingress_https_port="443"
+
+info() {
+    echo "$(tput bold)====> $1$(tput sgr0)"
+}
+
+info "Skip ingress installation."
+exit 0
```

Comment:
I had a bit of back and forth. Eventually went with NodePort Service. Now I finally understand how configuration this is used.